src/components/implementation/no_interface/llbooter/s_stub.S

```diff
@@ -1,4 +1,13 @@
 #include <cos_asm_server_stub_simple_stack.h>
 
 .text
+
+cos_asm_server_stub(fh_div_by_zero_err)
```

Comment:
This is part of why we should create a fault handler interface (or multiple).  The stubs should be shared between all components that implement the API